src/test/java/se/diabol/jenkins/pipeline/DeliveryPipelineViewTest.java

```diff
@@ -321,13 +320,35 @@ public void testGetItemsAndContainsWithFolders() throws Exception {
 
     }
 
+    @Test
+    public void testLimitedBuildsPipelines() throws Exception {
```

Comment:
Can you please change to a more descriptive method name? I would like to have "last job" included so if the test case fails it's easy to see what failed